### ISSUE TITLE
Do we still need to bundle Microsoft Redistributable packages?

### ISSUE THREAD

**Issue by manuelcortez** [manuelcortez/TWBlue#443]
Here at TWBlue, we have been shipping the software including the Microsoft Visual C++ redistibutable packages needed to run the application, so the app would work even if users don't have installed the microsoft packages. However, since we have moved to Python 3, and especially after we have started to use cx_freeze to bundle the application in a distributable file, I have been receiving a lot of reports regarding TWBlue being unable to be opened by users. Apparently this might have something to do with the fact that cx_freeze includes some DLL'S by itself, and those don't match the libraries we ship from our windows-dependencies repository. Unfortunately, this has never veen 100% reproducible, and while some people can have the application working perfectly in their machines, some other don't. This makes me wonder if do we still need to attempt to ship the Visual C++ distributable packages by ourselves, even if nowadays pretty much every system must have those packages in order for some app to work properly? to me, this looks no longer needed in our side. Mostly Windows 11/10 already include those packages by default, and users of Windows 8.1/8/7 are basically required to install one of those if they want many relatively modern apps to work properly. We can include a note in our downloads page, telling everyone which version of the package TWBlue would need to work properly and including a direct link. This way, we'd not be responsible for the many "TWBlue no longer works and gives many dll issues" caused by our attempts to bundle the packages by hand.
any thoughts?

**Comment by trypsynth** [comment]
Yeah, I vote to just own them from the package. It just makes the dists larger, and seems pointless for the reasons outlined above.
